Preferences.test.tsx

```diff
@@ -21,10 +21,10 @@ describe('Preferences', () => {
 
   it('updates after preferences are loaded', async () => {
     await AsyncStorage.setItem(PREFERENCES_KEY, JSON.stringify({center: 'BAC', hasSeenCenterPicker: true}));
-    const {result, waitForValueToChange} = renderHook(() => usePreferences(), {wrapper: PreferencesProvider});
+    const {result, waitForNextUpdate} = renderHook(() => usePreferences(), {wrapper: PreferencesProvider});
```

Comment:
using `waitForNextUpdate` is simpler and captures the spirit of what we're trying to do here. I made that change throughout.